app/components/data-download.tsx

```diff
@@ -321,6 +322,7 @@ const DownloadMenuItem = ({
 }) => {
   const locale = useLocale();
   const i18n = useI18n();
+  const client = useClient();
```

Comment:
It was related to storybook. In storybook, we have to execute against localhost:3000 and not against / (otherwise it tried localhost:6006/graphql). It was working already for useComponentsQuery etc... where the hook execute the query against the provider client, since in Storybook we provide a client correctly configured. But it was not working for functions using directly the imported client. Thus, the change :)

Reply:
Thanks for explaining 👌